src/legacy/core_plugins/kibana/public/management/sections/objects/lib/import_legacy_file.ts

```diff
@@ -17,10 +17,11 @@
  * under the License.
  */
 
-export async function importLegacyFile(file, FileReader = window.FileReader) {
+export async function importLegacyFile(file: File) {
```

Comment:
the `FileReader` param was only there for testing purposes. Adapted the tests and removed the param.

Reply:
src/legacy/core_plugins/kibana/public/management/sections/objects/lib/__jest__/import_legacy_file.test.js is still js file